### ISSUE TITLE
llvm 3.8 build problem

### ISSUE THREAD

**Issue by jpritikin** [ethereum/cpp-ethereum#2876]
I installed llvm from http://llvm.org/apt/trusty/ following
https://github.com/ethereum/cpp-ethereum/wiki/Building-on-Ubuntu

I checked out the master branch 9d118e62eb7 and tried to build,

````
cmake .. -DBUNDLE=minimal
make
````

It got stuck at,

````
[  0%] Built target BuildInfo.h
Linking CXX shared library libevmjit.so
/usr/bin/ld.gold: error: cannot find -lLLVMipa
collect2: error: ld returned 1 exit status
````

I can't find hte LLVMipa library. I see it in earlier versions of llvm, but not 3.8.

Please advise.

**Comment by CJentzsch** [comment]
We are looking at it. As a quick workaround for now you can disable evmjit which uses the llvm lib by doing `cmake .. -DBUNDLE=minimal -DEVMJIT=0`

**Comment by CJentzsch** [comment]
fixed by #2879 and #2878
